content/blog/2018-06-05-unconf18_recap_1.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: "unconf_recap_1"
+title: "Unconf18 projects 1: mchtoolbox, revtools, dataspice, rOpenSciEd, rOpenInterviews"
+date: 2018-06-05
+authors:
+  - name: Maëlle Salmon
+    url: http://www.masalmon.eu/
+categories: blog
+topicid: 
+tags:
+- community
+- ropensci
+- meetings
+- unconf
+---
+
+After Stefanie's general recap of [unconf 18](https://ropensci.org/blog/blog/2018/05/28/unconf2018), this week the blog will feature summaries of projects developed at the event: each day 4 to 5 projects will be highlighted. Later this year a handful of teams will post more thorough posts about their work. 
```

Comment:
After Stefanie's recap of [unconf18](https://ropensci.org/blog/blog/2018/06/06/unconf2018), this week the blog will feature brief summaries of projects developed at the event: each day 4 to 5 projects will be highlighted. In the following weeks, a handful of groups will share more thorough posts about their work.

Reply:
If your post goes up 6/6 mine needs to be 6/7 right?

Reply:
d'oh! I meant https://ropensci.org/blog/blog/2018/06/04/unconf18